frontend/app/pages/tasks/description/[id].vue

```diff
@@ -0,0 +1,39 @@
+<script setup lang="ts">
+import { FetchError } from 'ofetch'
+
+const description = useState('task-description', () => '')
+
+const route = useRoute()
+const task_id = String(route.params.id ?? 'simple-task-example')
+
+if (import.meta.server) {
```

Comment:
I think that now if you navigate to the other page, and back here it won't load the data (Because this code is not run on client side navigation